src/m365/spo/commands/roledefinition/roledefinition-list.ts

```diff
@@ -0,0 +1,67 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import { validation } from '../../../../utils';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+}
+
+class SpoRoleDefinitionListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.ROLEDEFINITION_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists of role definitions from web.';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['Id', 'Name'];
+  }
+
+  public commandAction(logger: Logger, args: CommandArgs, cb: () => void): void {
+    if (this.verbose) {
+      logger.logToStderr(`Getting role definitions list from ${args.options.webUrl}...`);
+    }
+
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/web/roledefinitions`,
+      headers: {
+        'accept': 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    request
+      .get<any>(requestOptions)
```

Comment:
The return type for the request shouldn't be `any` but rather `value: any[]` so that in the res property we get the known `value` array.